remoting/subsystem/src/main/java/org/jboss/as/remoting/RemotingSubsystemRootResource.java

```diff
@@ -227,10 +226,8 @@ private Attributes(ProcessType processType, boolean forDomain) {
                             .build();
                 }
             } else if (forDomain) {
-                this.worker = SimpleAttributeDefinitionBuilder.create(WORKER).setCapabilityReference((CapabilityReferenceRecorder) null).build();
```

Comment:
@pferraro Fyi re https://issues.redhat.com/browse/WFCORE-6473 and https://issues.redhat.com/browse/WFCORE-6472